CMSgov/design-system
# [WNMGDS-669] Update sketch to use color variables!

### Added
- Now using color variables in our Sketch file


![Screen Shot 2020-10-21 at 3 01 15 PM](https://user-images.githubusercontent.com/1449852/96770303-46eb6000-13ae-11eb-9afe-bdc05dd1ef61.png)

### How to test
1. Download the Sketch file from this PR here https://github.com/CMSgov/design-system/blob/3a1059f9285fcdfdf9ee7eb3f9c1dda018e981ee/design-assets/CMS-Design-System-UI-kit.sketch
1. See that color variables have been set up https://www.sketch.com/docs/styling/#color-variables
